bigtable/client/table.cc

```diff
@@ -58,48 +58,50 @@ void Table::Apply(SingleRowMutation&& mut) {
   // Build the RPC request, try to minimize copying.
   btproto::MutateRowRequest request;
   request.set_table_name(table_name_);
-  request.set_row_key(std::move(mut.row_key_));
-  request.mutable_mutations()->Swap(&mut.ops_);
+  mut.MoveTo(request);
+  //            request.set_row_key(std::move(mut.row_key()));
```

Comment:
Ahh, that explains my previous question, SGTM, but please remove the commented out code.